Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.yml

```diff
@@ -49,11 +52,42 @@ configuration:
   name: key
   required: false
   type: 14
-- display: Username
-  additionalinfo: Enable basic authentication for accessing the EDL. If empty, no authentication is enforced. 
+- additionalinfo: Enable basic authentication for accessing the EDL. If empty, no
+    authentication is enforced.
+  display: Username
   name: credentials
   required: false
   type: 9
+- additionalinfo: For use with mgw format to indicate the list type.
```

Comment:
fix this and consider using McAfee Web Gateway (mwg)

Reply:
done